### PR TITLE
Add SvgSprite

### PR DESCRIPTION
**Pull request by andyduong1920**
https://github.com/nimblehq/elixir-templates/issues/103

## What happened

[Describe the big picture of your changes here to communicate to the team why we should accept this pull request.](https://github.com/nimblehq/elixir-templates/issues/103)

## Insight

- Add `svg-sprite` and `svg-sprite.generate-icon` script into `package.json`
- Add the IconHelper module
- Import #{web_module}.IconHelper to the Web entrypoint
- Add the `Icon-Sprite` into Github Wiki.

## Proof Of Work

The test is passed

### CODE REVIEW

File: priv/templates/nimble_template/.github/wiki/Icon-Sprite.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Icon Sprite
```
Comment (rosle) [inline priv/templates/nimble_template/.github/wiki/Icon-Sprite.md:1]: Should we skip this heading, seems the wiki page title is already there, will this duplicate with that title 🤔

File: priv/templates/nimble_template/test/otp_app_web/helpers/icon_helper_test.exs.eex [hunk 2]
```diff
@@ -0,0 +1,19 @@
+defmodule <%= web_module %>.IconHelperTest do
+  use <%= web_module %>.ConnCase, async: true
+
+  import Phoenix.HTML, only: [safe_to_string: 1]
+
+  alias <%= web_module %>.IconHelper
+
+  describe "icon_tag/2" do
```
Comment (rosle) [inline priv/templates/nimble_template/test/otp_app_web/helpers/icon_helper_test.exs.eex:8]: Should it be
```suggestion
  describe "icon_tag/3" do
```

File: priv/templates/nimble_template/.github/wiki/Icon-Sprite.md (hunk 1 above)
Comment (andyduong1920) [reply priv/templates/nimble_template/.github/wiki/Icon-Sprite.md:1]: Yeah, correct @rosle 👍I will update this

Comment (andyduong1920) [reply priv/templates/nimble_template/.github/wiki/Icon-Sprite.md]: updated in https://github.com/nimblehq/elixir-templates/pull/146/commits/5db504f70b06726df8efe19e74b4abd1e173611f

File: priv/templates/nimble_template/test/otp_app_web/helpers/icon_helper_test.exs.eex (hunk 2 above)
Comment (andyduong1920) [reply priv/templates/nimble_template/test/otp_app_web/helpers/icon_helper_test.exs.eex]: updated in https://github.com/nimblehq/elixir-templates/pull/146/commits/5db504f70b06726df8efe19e74b4abd1e173611f @rosle

**Review by bterone — APPROVED** [review]

**Review by rosle — APPROVED** [review]

**Review by bterone — APPROVED** [review]

**Pull request merged**